src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* body {
+  background-color: lightgray;
+} */
+
+.header {
+  /* margin: 10px 50px; */
+  padding: 0 50px;
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header_logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header_logo img {
+  max-width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__link {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. In case 'Roboto' isn't available, it's good to have an alternative font-family specified.